client/README.md

```diff
@@ -2,13 +2,32 @@
 
 ## Getting Started
 
-Follow flutter installation instructions [here](https://flutter.dev/docs/get-started/install).
+NOTE: We use Flutter SDK version `1.x.x`, NOT version `2.x.x`!
 
-Clone the repo then:
+Install the latest `v1.x.x` Flutter release by downloading it [here](https://flutter.dev/docs/development/tools/sdk/releases) and then following the installation instructions [here](https://flutter.dev/docs/get-started/install).
+
+Clone the repo, then...
+
+### Set up Android Studio project
+
+- Open Android Studio and select "Open Project".
+- Select `app/client/app`. Wait a moment while Gradle syncs; this sync will **fail**.
+- Open `local.properties` and add entries like the following:
+
+```
+flutter.sdk=/path/to/your/flutter
+flutter.versionCode=1
+flutter.versionName=0.0.1
+```
+
+- Run `File -> Sync Project with Gradle Files` to re-sync; this sync will succeed.
+- Start your Android Virtual Device (AVD): `Tools` -> `AVD Manager` -> (on listed device) `Actions` -> `Run`.
+
+You can now run the app by running:
 
 ```
 cd client
-flutter run
+flutter run --flavor prod
```

Comment:
For development purposes, users should use the staging version in order to allow us access to the server's debug logs.
```suggestion
flutter run --flavor stag
```